app/lib/submission_builder/ty2022/states/ny/documents/state1099_g.rb

```diff
@@ -12,27 +12,27 @@ def document
               build_xml_doc("State1099G", documentId: "State1099G-#{form1099g.id}") do |xml|
                 if form1099g.payer_name && form1099g.payer_name != ''
                   xml.PayerName payerNameControl: form1099g.payer_name.gsub(/\s+/, '').upcase[0..3] do
-                    xml.BusinessNameLine1Txt form1099g.payer_name[0..74]
+                    xml.BusinessNameLine1Txt form1099g.payer_name&.strip.gsub(/\s+/, ' ').slice(0..74)
```

Comment:
I would prefer to use `0, 75` instead of `0..74` since you can see the length will be 75. Not urgent though I guess.

Reply:
also the `strip` should be after `slice` so that trailing spaces are removed if the slice end with a space.